.circleci/config.yml

```diff
@@ -134,6 +134,12 @@ workflows:
           executor:
             name: ci-images-large
           <<: *kind_configuration_helm3
+      - rok8s11/kubernetes_e2e_tests:
```

Comment:
Instead of starting a whole new CircleCI job, I think we should leverage the existing test script. That way the fleet install runs on each version of Kubernetes we support, not just 1.21

IIRC you mentioned that just adding another YAML file to `stable/insights-agent/ci` caused `ct` to run a separate test with that file - is that true?

If not maybe we can modify [this script](https://github.com/FairwindsOps/charts/blob/master/scripts/e2e-test.sh) to support it

Reply:
@rbren that's correct. If I add a *-values.yaml to the `stable/insights-agent-ci` folder, it will run the regular ct tests. Basically it will install a previous version of the chart with those *values.yaml and upgrade it to the new version. Then just install the new version.

My concerns are:
* The cluster name will in that case be static
* We can't do a check to make sure the be-main server is ready
* We can't do the kubectl wait for completion checks -> Not until later when I figure out how to do this

Is that what we want? 
I can still add this script to be run as part of the `e2e` tests so it runs for every kubernetes cluster. But it won't be through charts-testing.

Reply:
👍 makes sense! I like the solution you have here.